ui/common/src/main/res/values/strings.xml

```diff
@@ -269,6 +269,7 @@
   <string name="transaction_type_challenge_reward">Challenge Reward</string>
   <string name="transaction_type_challenge_reward_rejected">Rejected Challenge Reward</string>
   <string name="transaction_type_challenge_reward_reverted">Reverted Challenge Reward</string>
+  <string name="transaction_type_challenge_reward_reverted_date">The Challenge Reward of %s has been reverted.</string>
```

Comment:
Wrong String added. Correct one: `This Bonus has been reverted. For more information contact support`